guacamole/src/main/webapp/translations/zh.json

```diff
@@ -3,7 +3,6 @@
     "NAME" : "简体中文",
     
     "APP" : {
-
```

Comment:
I think this blank line should remain there and not be removed - the rest of the sections (below) and other translations have these blank lines for readability.

Reply:
@XYUU - If you can add back this poor space, I think we can move forward with the merge.

Reply:
Okay, I'll take the time to add these spaces.